src/streams/StreamScreen.js

```diff
@@ -22,22 +25,40 @@ class StreamScreen extends PureComponent<Props> {
     actions.navigateToEditStream(navigation.state.params.streamId);
   };
 
+  static contextTypes = {
+    styles: () => null,
+  };
+
+  toggleStreamPushNotification = () => {
+    const { subscriptions, navigation, auth } = this.props;
+    const { streamId } = navigation.state.params;
+    const subscription = subscriptions.find(x => x.stream_id === streamId) || NULL_SUBSCRIPTION;
+    toggleStreamNotification(auth, streamId, !subscription.push_notifications);
+  };
+
   render() {
     const { streams, subscriptions, navigation } = this.props;
     const { streamId } = navigation.state.params;
     const stream = streams.find(x => x.stream_id === streamId);
-    const subscription = subscriptions.find(x => x.stream_id === streamId);
+    const subscription = subscriptions.find(x => x.stream_id === streamId) || NULL_SUBSCRIPTION;
 
     return (
       <Screen title="Stream" padding>
         <StreamCard stream={stream} subscription={subscription} />
         <ZulipButton text="Edit" onPress={this.handleEdit} />
+        <OptionRow
+          label="Push notifications"
```

Comment:
Just use `Notifications`, I am intentionally using the shorter, simpler term (and no need to translate it then)